streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -67,12 +67,12 @@ object Checkpoint extends Logging {
   val REGEX = (PREFIX + """([\d]+)([\w\.]*)""").r
 
   /** Get the checkpoint file for the given checkpoint time */
-  def checkpointFile(checkpointDir: String, checkpointTime: Time) = {
+  def checkpointFile(checkpointDir: String, checkpointTime: Time): Path = {
```

Comment:
This isnt a public method. Are you referring to the fact that in Java bytecode it is public?